tests/MartinGeorgiev/Doctrine/DBAL/Types/TextArrayTest.php

```diff
@@ -46,11 +46,16 @@ public function validTransformations(): array
             ],
             [
                 'phpValue' => [
+                    1,
+                    '2',
+                    'text',
                     'some text here',
                     'and some here',
-                    'even here there is text',
+                    "single-quotes ain't no worry, right Alexander O'Vechkin?",
+                    'back-slashing\hooking though',
+                    'and "double-quotes"',
                 ],
-                'postgresValue' => '{"some text here","and some here","even here there is text"}',
+                'postgresValue' => '{1,2,"text","some text here","and some here","single-quotes ain\'t no worry, right Alexander O\'Vechkin?","back-slashing\\\hooking though","and \"double-quotes\""}',
```

Comment:
I believe the bit with `1,2,` is not valid as part of a text array in PostgreSql.

Reply:
Just added extra coverage to ilustrate that this is though what master code already does when facing numeric:
```
if (\is_numeric($text) || \ctype_digit($text)) {
                    $escapedText = $text;
```
from PostgreSql side also this works: word instead of "word": 
```
select unnest($${1,2,text,"some text here","and some here","single-quotes ain't no worry, right Alexander O'Vechkin?","back-slashing\hooking though","and \"double-quotes\""}$$::varchar[]);
```